README.md

```diff
@@ -1,121 +1,189 @@
+yarn add @lomray/react-native-responsive
 ```
 
-## Helper functions
-| Function name | Types | Description                                                                                                                                                             |
-|:--------------|:-----:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| isIOS         | `boolean` | Check if platform is IOS                                                                                                                                                |
-| isAndroid     | `boolean` | Check if platform is Android                                                                                                                                            |
-| wp            | `(widthPercent: number, disableRatio = false) => number` | Converts provided width percentage to independent pixel (dp). Disable ratio is intended for some cases when the width on tablets should be the full width of the screen |
-| hp            | `(heightPercent: number) => number` | Converts provided height percentage to independent pixel (dp).                                                                                                          |
-| fs            | `(fontPercent: number) => number` | Converts provided font size percentage to independent pixel (dp). Calculates based on the current screen width (including orientation)                                  |
-| orIsP         | `boolean` | Check if orientation is portrait                                                                                                                                        |
-| orIsL         | `boolean` | Check if orientation is landscape                                                                                                                                       |
-| lor           | `(that: Component, callback: (val: Component) => EmitterSubscription` | Event listener function that detects orientation change                                                                                                                 |
-| makePadding  | `(top: number, right?: number, bottom?: number, left?: number) => ({ paddingTop, paddingRight, paddingBottom, paddingLeft })` | Make paddings                                                                                                                                                           |
+## How to use
+Initialize ResponsiveManager with your device parameters by design to get helper functions.
+```typescript
+/**
+ * src/services/responsive-manager.ts
+ */
+import { ResponsiveManager } from '@lomray/react-native-responsive';
 
-## Basic example
+const { fs, hp, wp } = new ResponsiveManager({ height: 844, width: 390 });
+
+const createStyles = ResponsiveManager.createStyles;
+
+export { createStyles, fs, hp, wp };
```

Comment:
createStyles is static, because I needed it at useStyles hook. So it’s better to put it in a separate file, or this option

Reply:
ok. In this case move createStyles in separate file

Reply:
I can't do "export new ResponsiveManager" because "TS error: declaration expected". And one export should be by default (by our ESlint rules). It seems to me that it is more convenient to leave it as is